qemu/qmp.go

```diff
@@ -750,6 +759,11 @@ func (q *QMP) ExecuteBlockdevAdd(ctx context.Context, device, blockdevID string)
 	return q.executeCommand(ctx, "blockdev-add", args, nil)
 }
 
+// ExecuteBlockdevAdd sends a blockdev-add to the QEMU instance.
+func (q *QMP) ExecuteBlockdevAdd(ctx context.Context, device, blockdevID string) error {
+	return q.ExecuteBlockdevAddWithCache(ctx, device, blockdevID, false, false)
```

Comment:
By doing so, we assume that QEMU always knows about the parameters `direct` and `no-flush`. Is this true?
I want to make sure we don't introduce any regression by adding those new parameters, even when they're not strictly needed.
If this introduce regression, we should simply duplicate those functions I think.

Reply:
https://github.com/qemu/qemu/blob/master/qapi/block-core.json said they are "Since: 2.9".

Reply:
@markdryan I don't know what you think about this, but I would recommend creating a new function here. It'd be better to prevent any potential regression for versions older than 2.9, WDYT?

Reply:
@sboeuf Good catch.  I think you're right about this.  @teawater Could you keep the two functions separate?  It was a nice idea to try and re-use the existing code but as @sboeuf points out it might cause problems with older versions of qemu.